src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/DiagnosticsTelemetryModule.cs

```diff
@@ -43,6 +42,16 @@ public DiagnosticsTelemetryModule()
         }
 
         /// <summary>
+        /// Gets or sets the delay between heartbeat emmissions in milliseconds.
+        /// </summary>
+        public int MillisecondsBetweenHeartbeats { get; set; }
+
+        /// <summary>
+        /// Gets or sets property names that are allowed to be sent along with the health heartbeats emitted. * means allow all default properties through.
```

Comment:
this comment should list known properties. Alternatively you can specify them in a remarks section

Reply:
Added TODO and remarks section. Will fill it in as I add properties.